src/renderer/components/+namespaces/namespace.store.ts

```diff
@@ -103,8 +103,20 @@ export class NamespaceStore extends KubeObjectStore<Namespace> {
     return [];
   }
 
-  public getContextNamespaces(): string[] {
-    return Array.from(this.contextNs);
+  getContextNamespaces(): string[] {
```

Comment:
This function isn't just getting the namespaces listed in the context, its name should be changed

Reply:
Not really. It differs from `this.contexNs` list. Return full namespaces when for example nothing selected (aka "All namespaces")

Reply:
Yes I know it differs from that list, but it isn't really just getting that list (which is what I would guess from the name).

How about something like `getCurrentSelectedNamespaces()`?

Reply:
Sounds good. But it more aligns with more context meaning of namespaces over just selection.